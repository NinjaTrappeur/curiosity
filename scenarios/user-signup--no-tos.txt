reset
# Shows that --accept-tos must be provided
user signup alice secret alice@example.com
