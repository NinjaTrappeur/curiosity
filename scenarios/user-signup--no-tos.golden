1: reset
State is now empty.
3: user signup alice secret alice@example.com
NoTosConsent
